### PR TITLE
Off-policy training tutorial

### PR DESCRIPTION
**Pull request by jordis-ai2**
(and some minor fixes)

### CODE REVIEW

**Review by anikem — APPROVED** [review]

**Review by Lucaweihs — COMMENTED** [review]
Nice, added some minor comments.

**Comment by jordis-ai2** [comment]
I ended up changing a couple more things in the training pipeline and accompanying text...

File: docs/tutorials/offpolicy-tutorial.md [hunk 1]
```diff
@@ -149,10 +151,10 @@ For the experiment configuration, we'll build on top of an existing
...
 The complete `ExperimentConfig` file for off-policy training is
 [here](/api/projects/tutorials/babyai_go_to_local_bc_offpolicy/#bcoffpolicybabyaigotolocalexperimentconfig), but let's
-focus on the most relevant aspects to enable off-policy training. 
-Off-policy training involves instantiating
-[OffPolicyPipelineComponents](/api/utils/experiment_utils/#offpolicypipelinecomponent)
-and passing them to the current pipeline stage.
+focus on the most relevant aspect to enable this type of training: 
+We just need to instantiate a `TrainingPipeline` with an
+[OffPolicyPipelineComponent](/api/utils/experiment_utils/#offpolicypipelinecomponent)
+where required in our `PipelineStage` defined in the `training_pipeline` method.
```
Comment (Lucaweihs) [inline docs/tutorials/offpolicy-tutorial.md:157]: I think there is an error here. I would change this sentence from "We just need to instantiate a `TrainingPipeline` with an..." to:

"providing a [OffPolicyPipelineComponent](/api/utils/experiment_utils/#offpolicypipelinecomponent) object as input to a `PipelineStage` when instantiating the `TrainingPipeline`."

File: docs/tutorials/offpolicy-tutorial.md [hunk 2]
```diff
@@ -172,40 +174,50 @@ class BCOffPolicyBabyAIGoToLocalExperimentConfig(BaseBabyAIGoToLocalExperimentCo
...
-        total_train_steps = cls.TOTAL_IL_TRAIN_STEPS
-        ppo_info = cls.rl_loss_default("ppo", steps=-1)
-
-        num_mini_batch = ppo_info["num_mini_batch"]
-        update_repeats = ppo_info["update_repeats"]
-
-        return cls._training_pipeline(
-            # Instantiate the MiniGridOffPolicyExpertCELoss
+        total_train_steps = int(1e7)
+        num_steps=128
```
Comment (Lucaweihs) [inline docs/tutorials/offpolicy-tutorial.md:178]: `num_steps=128` -> `num_steps = 128`

**Review by Lucaweihs — COMMENTED** [review]
One tiny comment, feel free to merge after looking at it.

**Comment by jordis-ai2** [comment]
Merging after updating...

**Pull request merged**